### ISSUE TITLE
release v1.3.7 causing build issues in rhel7 docker image

### ISSUE THREAD

**Issue by codedawi** [mehcode/python-xmlsec#141]
With the new release of `1.3.7` we are seeing build/compiler errors in our rhel7 docker container. We needed to install `xmlsec` as dependency for `python3-saml` but the patch version bump caused issues: 

```
Building wheel for xmlsec (PEP 517): started
  Building wheel for xmlsec (PEP 517): finished with status 'error'
  ERROR: Command errored out with exit status 1:
   command: /opt/bb/bin/python3.6 /opt/bb/lib/python3.6/site-packages/pip/_vendor/pep517/_in_process.py build_wheel /tmp/tmpp5fne7g4
       cwd: /tmp/pip-install-lhs78mrq/xmlsec
  Complete output (62 lines):
  running bdist_wheel
  running build
  running build_py
  package init file 'src/xmlsec/__init__.py' not found (or not a regular file)
  creating build
  creating build/lib.linux-x86_64-3.6
  creating build/lib.linux-x86_64-3.6/xmlsec
  copying src/xmlsec/py.typed -> build/lib.linux-x86_64-3.6/xmlsec
  copying src/xmlsec/constants.pyi -> build/lib.linux-x86_64-3.6/xmlsec
  copying src/xmlsec/template.pyi -> build/lib.linux-x86_64-3.6/xmlsec
  copying src/xmlsec/tree.pyi -> build/lib.linux-x86_64-3.6/xmlsec
  copying src/xmlsec/__init__.pyi -> build/lib.linux-x86_64-3.6/xmlsec
  running build_ext
  building 'xmlsec' extension
  creating build/temp.linux-x86_64-3.6
  creating build/temp.linux-x86_64-3.6/tmp
  creating build/temp.linux-x86_64-3.6/tmp/pip-install-lhs78mrq
  creating build/temp.linux-x86_64-3.6/tmp/pip-install-lhs78mrq/xmlsec
  creating build/temp.linux-x86_64-3.6/tmp/pip-install-lhs78mrq/xmlsec/src
  /opt/bb/bin/gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -m64 -march=westmere -fPIC -DXMLSEC_CRYPTO=openssl -DXMLSEC_CRYPTO_DYNAMIC_LOADING=1 -D__XMLSEC_FUNCTION__=__FUNCTION__ -DXMLSEC_NO_SIZE_T -DXMLSEC_NO_GOST=1 -DXMLSEC_NO_XKMS=1 -DXMLSEC_DL_LIBLTDL=1 -DMODULE_NAME=xmlsec -DMODULE_VERSION=1.3.7 -I/opt/bb/include/xmlsec1 -

......

  rpm: /opt/bb/lib64/liblzma.so.5: no version information available (required by /lib64/librpmio.so.3)
  rpm: /opt/bb/lib64/liblzma.so.5: no version information available (required by /lib64/librpmio.so.3)
  /tmp/pip-install-lhs78mrq/xmlsec/src/main.c: In function 'PyXmlSec_GetCryptoLibName':
  <command-line>:0:15: error: 'openssl' undeclared (first use in this function)
  /tmp/pip-install-lhs78mrq/xmlsec/src/main.c:33:49: note: in expansion of macro 'XMLSEC_CRYPTO'
       const xmlChar* cryptoLib = (const xmlChar*) XMLSEC_CRYPTO;
                                                   ^~~~~~~~~~~~~
  <command-line>:0:15: note: each undeclared identifier is reported only once for each function it appears in
  /tmp/pip-install-lhs78mrq/xmlsec/src/main.c:33:49: note: in expansion of macro 'XMLSEC_CRYPTO'
       const xmlChar* cryptoLib = (const xmlChar*) XMLSEC_CRYPTO;
                                                   ^~~~~~~~~~~~~
  error: command '/opt/bb/bin/gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for xmlsec
```
We resolved this by installing `xmlsec==1.3.3` prior to pip installing `python3-saml` package.

**Comment by codedawi** [comment]
Same as #140 put in a minute before this issue.

**Comment by hoefling** [comment]
Sigh. I knew I will forget something. @codedawi can you please try installing the fixed version and check whether the error is resolved? I have made an upload to TestPyPI, you can try it out via

```sh
$ pip install xmlsec==1.3.8a1 --extra-index-url=https://test.pypi.org/simple
```

If you can confirm the issue resolved, I will prepare another release later today.

**Comment by hoefling** [comment]
I have released [1.3.8](https://pypi.org/project/xmlsec/1.3.8/) which should resolve the issue. Sorry for the noise!

**Comment by codedawi** [comment]
Thank you! @hoefling
